src/argilla/training/base.py

```diff
@@ -0,0 +1,163 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import inspect
+import logging
+from typing import List, Union
+
+import argilla as rg
+from argilla.training.setfit import ArgillaSetFitTrainer
+
+
+class ArgillaBaseTrainer(object):
```

Comment:
```suggestion
class ArgillaTrainer:
```
No need to specify that it is a Base trainer. If there's no qualifier (`ArgillaSetFitTrainer`, `ArgillaSpaCyTrainer`, `ArgillaTransformersTrainer`), then we already know that it's the "base" one.
Also, for Python 3 we don't need to subclass `object` anymore.

Reply:
Thanks, resolved this.